Generate Keys.txt

```diff
@@ -0,0 +1,23 @@
+# Generate Keys
```

Comment:
if this is a manual step can we update the Readme?

Reply:
This is not needed for the changed implementation

Reply:
removed